### DISCUSSION TITLE
Cannot connect with my cell phone in local network.

### DISCUSSION THREAD

**Discussion by timeanddoctor** [Kitware/trame#188] [General]
And to prevent your browser from opening, add '--server' to your command line.

App running at:
 - Local:   http://localhost:8080/
 - Network: http://192.168.10.10:8080/

Note that for multi-users you need to use and configure a launcher.

After the up message, I used my cell phone to connect but failed.
Are there any help tutorl for this step. 
Thanks.

---------------------------
windows 10 ,python3.6,I can succeed connect with other Androi screen share App in local network for my computer.

**Comment by jourdain** [comment]
It should work if your phone is on the same network as your computer. What is the issue/error?
